### PR TITLE
DTSPB-2753 DOB equal DOD Application for probate practitioner and cas…

### PR DESCRIPTION
**Pull request by MarcusHMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-2753
https://tools.hmcts.net/jira/browse/DTSPB-2754

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/validator/DobDodValidationRule.java [hunk 1]
```diff
@@ -47,10 +47,6 @@ public List<FieldErrorResponse> validate(CCDData ccdData) {
                         codes.add(CODE_DOD_IN_FUTURE);
                     }
 
-                    if (dod.equals(dob)) {
-                        codes.add(CODE_DOD_ON_DOB);
```
Comment (nigeldunne) [inline src/main/java/uk/gov/hmcts/probate/validator/DobDodValidationRule.java:51]: Delete the String constant

File: src/functionalTest/resources/json/success.solicitorCreate.json [hunk 2]
```diff
@@ -95,7 +95,7 @@
         }
       ],
       "deceasedDomicileInEngWales": "Yes",
-      "ihtNetValue": "1000000",
```
Comment (nigeldunne) [inline src/functionalTest/resources/json/success.solicitorCreate.json:98]: Is this intentional?

Comment (MarcusHMCTS) [reply src/functionalTest/resources/json/success.solicitorCreate.json:98]: yes, originally ihtNetValue is 1000000 which is greater than "ihtGrossValue": "100001"
and i got this error
{"data":null,"errors":["The gross probate value cannot be less than the net probate value"],"warnings":null}
so i change it to 100000

File: src/main/java/uk/gov/hmcts/probate/validator/DobDodValidationRule.java (hunk 1 above)
Comment (MarcusHMCTS) [reply src/main/java/uk/gov/hmcts/probate/validator/DobDodValidationRule.java:51]: Done, thanks

**Review by nigeldunne — APPROVED** [review]

**Pull request merged**